app/_src/gateway/install/index.md

```diff
@@ -1,14 +1,25 @@
 ---
 title: Installation Options
+subtitle: Kong is a low-demand, high-performing API gateway. You can setup Kong with Konnect or it can be installed on many different self-managed systems.
```

Comment:
```suggestion
subtitle: Kong Gateway is a low-demand, high-performing API gateway. You can set up Kong Gateway with Konnect, or install it on various self-managed systems.
```
I know that "Kong" was in the mockups, but we really should avoid using the company name as the Gateway name in the docs.